mergify/datadog_checks/mergify/__about__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.1"
+__version__ = "0.0.2"
```

Comment:
nit - I would have bump the minor version instead of the patch one since you add a functionality that is backward compatible and not a fix. However, feel free to ignore my comment, how you manage the version number in your integration is completely up to you.

Reply:
You're right, my first time manually bumping the version of something ˆˆ Modified it, thanks !